components/layout/MFooter.vue

```diff
@@ -79,7 +79,7 @@ export default {
 }
 
 footer {
-  padding: 2rem 0 2rem 0;
+  padding: 2rem 1rem;
```

Comment:
This causes the footer to be very weirdly aligned on the mods pages, dashboard pages, and any other page that uses a thinner footer, on desktop
![image](https://i.imgur.com/aGT4uqN.png)

Reply:
My bad, I should have noticed this. I was trying to fix the footer on the index page.

Fixed in aa315876d65c10bc276fe71942e03101481c22a3 using a new prop just for padding.